src/ray/common/bundle_spec.cc

```diff
@@ -203,6 +203,11 @@ std::unordered_map<std::string, double> AddPlacementGroupConstraint(
   auto bundle_key =
       FormatPlacementGroupResource(kBundle_ResourceLabel, placement_group_id, -1);
   new_resources[bundle_key] = 0.001;
+  if (bundle_index >= 0) {
```

Comment:
Actually I'm wondering if we should only add bundle resource when normal resources are empty.

Reply:
We should add index resource in case users want to use bundle_index